lib/serialisers/json-0.6.js

```diff
@@ -24,21 +24,23 @@ module.exports = createClass({
       payload['meta'] = this.serialiseObject(element.meta);
     }
 
-    if (element.attributes.length > 0) {
-      if (this[element.element + 'SerialiseAttributes']) {
-        payload['attributes'] = this[element.element + 'SerialiseAttributes'](element);
-      } else {
-        var attributes = element.attributes;
-
-        // Meta attribute was renamed to metadata
-        if (attributes.get('metadata')) {
-          attributes = attributes.clone();
-          attributes.set('meta', attributes.get('metadata'));
-          attributes.remove('metadata');
-        }
+    if (this[element.element + 'SerialiseAttributes']) {
```

Comment:
Since the custom `SerialiseAttributes` handler may add attributes even if there we're non already we shouldn't check for `element.attributes.length > 0` first.